app/controllers/InscriptionController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Application\Controllers\InscriptionController;
+
+use Application\Core\Database\Database\DatabaseConnection;
+use Application\Model\UserModel;
+use Application\Model\UserModel\UsersRepository;
+
+use Application\Controllers\Controller;
+use Application\Controllers\connexion\ConnexionController;
+
+/**
+ * Class of user inscription
+ */
+class InscriptionController extends Controller
+{
+  /**
+   * function just to go the inscription page
+   *
+   * @return void
+   */
+  public function inscription()
+  {
+    $this->twig->display('inscription/inscription.html.twig');
+  }
+
+  /**
+   * function to create a user account (with crypt password)
+   *
+   * @return void
+   */
+  public function CreateAccount(array $postData)
```

Comment:
```suggestion
  public function createAccount(array $postData)
```

function -> camelCase